server/views/macros/tasks-details.njk

```diff
@@ -8,18 +8,30 @@
       </h3>
         </header>
         <div class="app-summary-card__body">
-          <div class="govuk-form-group govuk-!-margin-0">
-            <input type="hidden" id="updated" name="updated" value="today">
-            <input type="hidden" id="pathway" name="pathway" value="attitudes">
-            <a id="generate-pdf-task-btn"  class="govuk-button govuk-button--primary" href="/print/packPdf?prisonerNumber={{prisonerNumber}}" target="_blank">
-          Plan Your Future pack PDF
-        </a>
-          </div>
-          <div class="govuk-form-group govuk-!-margin-0">
-            <a id="generate-otp-task-btn" class="govuk-button govuk-button--primary" href="/print/otp?prisonerNumber={{prisonerNumber}}">
-              Generate First-time ID code
-            </a>
-          </div>
+          {% if watchlistViewEnabled == true %}
+            <div class="govuk-form-group govuk-!-margin-0">
+              <form action="/prisoner-overview/" method="GET" novalidate>
```

Comment:
interesting, usually on forms I need to add
`<input type="hidden" name="_csrf" value="{{ csrfToken }}"/>`
Does this work without csrf token?

Reply:
ah I think it's the novalidate, what happens it you remove novalidate?